### PR TITLE
## React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by abiodunraheem**
## React capstone project - Metrics webapp

- Used React components.
- Used React props.
- Used React Router.
- Connected React and Redux.
- Handle events in a React app.
- Wrote integration tests with a React testing library.
- Used styles in a React app.
- Used React life cycle methods.
- Applied React best practices and language style guides in code.
- Used store, actions and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+## Live video 
+[Live video](https://www.loom.com/share/7c7148b35edd44428e817e76219935b1)
+
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
```
Comment (Deepakdanger) [inline README.md:36]: - [ ] KIndly include code here also.
- `git clone https://github.com/abiodunraheem/covid-19-statistics.git`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+- CSS
+- JAVASCRIPT
+- React
+- Covid-19 API
+
+## Live Demo 
+[Live Demo Link](https://62bb0e3e5ee5341abc51b517--jolly-douhua-57a5de.netlify.app/)
+
+## Live video 
+[Live video](https://www.loom.com/share/7c7148b35edd44428e817e76219935b1)
```
Comment (Deepakdanger) [inline README.md]: - [ ] In the video, the mic is not working as it doesn't record your sound, Kindly make the video with a proper checkup once more.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @abiodunraheem, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] I think, you didn't understand the concept of using containers and components in react because you are breaking the fetch result at the fetching time. you can refer to this [blog](https://blog.openreplay.com/understanding-the-container-component-pattern-with-react-hooks) for understanding component.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @abiodunraheem 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Great work recording the video

ApprovedAgreeGIF


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**